.github/scripts/simple-smoke-test.sh

```diff
@@ -55,7 +94,7 @@ function install_clc() {
       echo "Retrying clc installation..."
       sleep 3
     done
-  export PATH=$PATH:$HOME/.hazelcast/bin
+  export PATH=${PATH}:${HOME}/.hazelcast/bin
   clc config add default cluster.name=dev cluster.address=localhost
```

Comment:
What is the reason for this?
Add comment pls as feels out of place otherwise

Reply:
> What is the reason for this? Add comment pls as feels out of place otherwise

What do you mean?
My change or the modifications to `PATH`?

Reply:
Ah ok I see what you mean.

`clc config add default cluster.name=dev cluster` is existing functionality, I'm not sure why it's there. It was added in 65f4b3fbe39a5ce6d978ff68803dc4d67090efd9 if that helps.

Reply:
thanks for confirming. resolving